### DISCUSSION TITLE
Checking maxheight quest

### DISCUSSION THREAD

**Discussion by NathanARF** [streetcomplete/StreetComplete#5036] [Ideas]
Would be good to have a quest similar to others that asks if a maxheight value is still the same after a couple of years. Potentially also useful for maxweight tags. Here in the UK it's not uncommon for a bridge to be reduced in height by a small amount after resurveys, or even raised in height sometimes to stop lorries striking them. This quest would also be useful for checking incorrect maxheight tags, as well as updating maxheight tags where the signage was recently upgraded from imperial only to metric + imperial. Could have a recheck duration of 2 years perhaps, with a longer duration where the value is `maxheight=default`, as in this case it is less likely the height would have reduced.

Also sorry if this has been asked before. I spent 10 minutes checking this time!

**Comment by matkoniecz** [comment]
Looking at https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/max_weight/AddMaxWeight.kt https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/max_height/AddMaxHeight.kt

It is not added yet, seems to make sense - but every 2 years seem significantly too often to me.

**Reply by mnalis** [reply]
> It is not added yet, seems to make sense - but every 2 years seem significantly too often to me.

I'd agree -  in Croatia `maxweight` / `maxheight` changes would too happen very rarely (if ever). It would be way too spammy with 2 years in 99% of the cases... Maybe every 10 years? (which can be then shortened to 5 ears by user selecting in preferences they want to be asked more often)
